src/views/workflow-summary-tab/workflow-summary-tab-details/workflow-summary-tab-details.tsx

```diff
@@ -0,0 +1,60 @@
+'use client';
+import React from 'react';
+
+import { LabelMedium, LabelXSmall } from 'baseui/typography';
+
+import useStyletronClasses from '@/hooks/use-styletron-classes';
+
+import workflowSummaryTabDetailsConfig from '../config/workflow-summary-tab-details.config';
+
+import { cssStyles } from './workflow-summary-tab-details.styles';
+import { type Props } from './workflow-summary-tab-details.types';
+
+export default function WorkflowSummaryTabDetails({
+  firstHistoryEvent,
+  lastHistoryEvent,
+  params,
+}: Props) {
+  const { cls } = useStyletronClasses(cssStyles);
+
+  return (
+    <div className={cls.pageContainer}>
+      <div>
+        <LabelMedium>
+          <strong>Workflow: </strong>
+          {
+            firstHistoryEvent?.workflowExecutionStartedEventAttributes
+              ?.workflowType?.name
+          }
+        </LabelMedium>
+      </div>
+      <div>
+        {workflowSummaryTabDetailsConfig
+          .filter(
+            (c) =>
+              !c.hide ||
+              !c.hide({
+                firstEvent: firstHistoryEvent,
+                lastEvent: lastHistoryEvent,
+                params,
+              })
+          )
+          .map((c) => (
+            <div className={cls.detailsRow} key={c.key}>
+              <LabelXSmall className={cls.detailsLabel}>
+                {c.getLabel()}
+              </LabelXSmall>
+              {/*supress hydration warning for dates */}
```

Comment:
I have the same issue with dates in the domain workflows table. Did you have any other fixes in mind?

Reply:
Instead of wrapping all components with suppressHydrationWarning, it would be better if we suppressed it only for dates